test/Libraries/RevitTestServices/RevitSystemTestBase.cs

```diff
@@ -84,6 +84,14 @@ private void SetDefaultValuesToUninitializedProperties()
         {
             var fi = new FileInfo(Assembly.GetExecutingAssembly().Location);
             string assDir = fi.DirectoryName;
+            
+            //It is used to get the path of journal file.
+            if (RevitSystemTestBase.IsJournalReplaying())
```

Comment:
We put each type of journal file, such as AdaptiveComponent, AnalysisDisplay, and their Revit files and dynamo files into Revit's Regression \ D4R folder. The results are Regression \ D4R \ AdaptiveComponent, Regression \ D4R \ AnalysisDisplay, The path to the file is different. We have to get where each journal file runs when Revit gets it, and then get the location of the required Revit and dynamo files.

Reply:
I think @mjkkirschner may mean that these comments added in the code?

Reply:
@ZiyunShang yes, please add this comment or something similar to the source code.